website/blog/2026/03/03-04-paving-the-way-for-a-new-logging-backend-the-victorialogsbackend-feature-gate.md

```
@@ -0,0 +1,44 @@
+newsSubtitle: March 04, 2026
+publishdate: 2026-03-04
+authors:
+- avatar: https://avatars.githubusercontent.com/rrhubenov
+  login: rrhubenov
+  name: rhubenov
+aliases: ["/blog/2026/03/04/paving-the-way-for-a-new-logging-backend-the-victorialogsbackend-feature-gate"]
+---
+
+As part of Gardener's ongoing effort to evolve its observability stack, outlined in [GEP-35](https://github.com/gardener/gardener/blob/master/docs/proposals/35-observability2.0-victoria-logs.md), the first step has been taken to integrate VictoriaLogs as the future log aggregation system, eventually replacing Vali. A new feature gate, `VictoriaLogsBackend`, has been introduced to facilitate a smooth, phased migration.
+
+### A New Feature Gate for a Gradual Transition
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)